processor/groupbytraceprocessor/storage_memory.go

```diff
@@ -0,0 +1,99 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package groupbytraceprocessor
+
+import (
+	"errors"
+	"sync"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+var errNilResourceSpans = errors.New("the provided trace is invalid (nil)")
+
+type memoryStorage struct {
+	sync.RWMutex
+	content map[string]pdata.ResourceSpans
+}
+
+var _ storage = (*memoryStorage)(nil)
+
+func newMemoryStorage() *memoryStorage {
+	return &memoryStorage{
+		content: make(map[string]pdata.ResourceSpans),
+	}
+}
+
+func (st *memoryStorage) createOrAppend(traceID pdata.TraceID, trace pdata.ResourceSpans) error {
+	if trace.IsNil() {
+		return errNilResourceSpans
+	}
+
+	hashedID := traceID.String()
```

Comment:
`sTraceID` here and in other places, you don't compute hash anymore.

Reply:
Have you forgotten to push some changes? Still named `hashedID`.

Reply:
Indeed, the storage was missing. This time I checked with `grep hash processor/groupbytraceprocessor -ir` and `hashID` isn't being used anymore.